libs/designer/src/lib/core/state/workflow/workflowSlice.ts

```diff
@@ -249,6 +252,24 @@ export const workflowSlice = createSlice({
         !!node?.children?.length && stack.push(...node.children);
       }
     },
+    updateNodeInfo: (state: WorkflowState, action: PayloadAction<{ nodes: Node[]; elkNodes?: ElkNode[] }>) => {
```

Comment:
@rllyy97 Passing both nodes and elkNodes since nodes here do not contain dimensions data. Since we are not using height/width, I wasn't sure how necessary this is for now. Alternatively, we can update this in the updateNodeSizes call, but that would be another huge loop so wasn't sure if we want that.

Reply:
Seems like height/width of the WorkflowNode is being used in `convertWorkflowGraphToElkGraph` in `elkLayout.ts`, if we would change the height/width being used here as ReactFlowNodeInfo, we can even get rid of the `updateNodeSizes ` - not sure if this is the direction we want to go to.

Reply:
I'm a little confused on what you're asking / proposing.  From our conversation yesterday let's leave the size logic the same as it was, and put the position logic into a new place in state (which it looks like you've done in your last commit).  Is what you have working at the moment?

Reply:
I am passing in an additional parameter (elkNodes) just to save the height/width into the new structure that is not currently being used anywhere. I was wondering if this still aligns with what we discussed